{"uuid":"21e2116e-0334-4c07-9cc1-fb1d090968b8","historyId":"48c77f544fde95f96fb8e753c898afa2","testCaseId":"bd30a3da7590ddee7efc1a26302e23ff","fullName":"src/test/resources/Feature/Arrays.feature:106","labels":[{"name":"host","value":"DESKTOP-TCJEROD"},{"name":"thread","value":"18876@DESKTOP-TCJEROD.main(1)"},{"name":"feature","value":"Arrays PracticeQuestion"},{"name":"story","value":"Verify that user receives Success message on submitting valid python code for Find Numbers with Even Number of Digits question"},{"name":"suite","value":"Arrays PracticeQuestion"},{"name":"testClass","value":"Verify that user receives Success message on submitting valid python code for Find Numbers with Even Number of Digits question"},{"name":"framework","value":"cucumber4jvm"},{"name":"language","value":"java"},{"name":"gherkin_uri","value":"file:///C:/Users/Kruthi/dsAlgo2025/Ds-Algo2025/src/test/resources/Feature/Arrays.feature"},{"name":"package","value":"src.test.resources.Feature.Arrays_feature.Arrays PracticeQuestion"}],"links":[],"name":"Verify that user receives Success message on submitting valid python code for Find Numbers with Even Number of Digits question","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"BUG: Expected message 'Submission Successful' but got: 'Error occurred during submission' expected [true] but found [false]","trace":"java.lang.AssertionError: BUG: Expected message 'Submission Successful' but got: 'Error occurred during submission' expected [true] but found [false]\r\n\tat org.testng.Assert.fail(Assert.java:111)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1590)\r\n\tat org.testng.Assert.assertTrue(Assert.java:57)\r\n\tat stepDefinitions.ArraysStep.the_user_gets_success_message_submission_successful_in_the_console(ArraysStep.java:288)\r\n\tat ✽.The user gets success message Submission successful in the console(file:///C:/Users/Kruthi/dsAlgo2025/Ds-Algo2025/src/test/resources/Feature/Arrays.feature:109)\r\n"},"stage":"finished","steps":[{"name":"Given The User is in home page after sign in","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1746984464408,"stop":1746984466180},{"name":"When The user scrolls down to the Data Structures dropdown and selects Array","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1746984466181,"stop":1746984467245},{"name":"Then User is navigated to Array Page","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1746984467245,"stop":1746984467293},{"name":"Given The user is on the Find Numbers practice question editor","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1746984467293,"stop":1746984472125},{"name":"When The user writes valid code from row number and clicks the Submit Button Find Numbers","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1746984472125,"stop":1746984472329},{"name":"Then The user gets success message Submission successful in the console","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"BUG: Expected message 'Submission Successful' but got: 'Error occurred during submission' expected [true] but found [false]","trace":"java.lang.AssertionError: BUG: Expected message 'Submission Successful' but got: 'Error occurred during submission' expected [true] but found [false]\r\n\tat org.testng.Assert.fail(Assert.java:111)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1590)\r\n\tat org.testng.Assert.assertTrue(Assert.java:57)\r\n\tat stepDefinitions.ArraysStep.the_user_gets_success_message_submission_successful_in_the_console(ArraysStep.java:288)\r\n\tat ✽.The user gets success message Submission successful in the console(file:///C:/Users/Kruthi/dsAlgo2025/Ds-Algo2025/src/test/resources/Feature/Arrays.feature:109)\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1746984472329,"stop":1746984472959}],"attachments":[],"parameters":[],"start":1746984462361,"stop":1746984473418}